Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -108,6 +106,7 @@ static void HandleFrameStepHotkeys()
 
   if (IsHotkey(HK_FRAME_ADVANCE, true))
   {
+    constexpr int FRAME_STEP_DELAY = 30;
```

Comment:
Others can comment, but I think I'd prefer a pair of very-related `constexpr` like this to be centrally located. I understand the general benefits of keeping variables in the smallest scope possible, but I don't know if I'd always apply that logic to `constexpr`s like this. I personally think the existing code is more readable.